### PR TITLE
bower.json: Upgrade bower dependencies

### PR DESCRIPTION
**Pull request by sahilarora535**
- Upgrade all dependencies in 'bower.json' to latest stable versions.
 - Update 'bootstrap' resolution to '^3.3.7'.

Closes https://github.com/coala/coala-html/issues/115

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalahtml/_coalahtml/bower.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-    "angular": "^1.4.0",
-    "angular-route": "^1.4.0",
-    "bootstrap": "^3.3.6",
-    "google-code-prettify": "^1.0.4",
-    "angular-sanitize": "^1.5.0",
-    "bootstrap-material-design": "^0.5.10"
+    "angular": "^1.6.2",
+    "angular-route": "^1.6.2",
+    "bootstrap": "^3.3.7",
+    "google-code-prettify": "^1.0.5",
```
Comment (jayvdb) [inline coalahtml/_coalahtml/bower.json]: this change isnt necessary, as 1.0.5 is compatible with 1.0.4

File: coalahtml/_coalahtml/bower.json [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
   "dependencies": {
-    "angular": "^1.4.0",
-    "angular-route": "^1.4.0",
-    "bootstrap": "^3.3.6",
-    "google-code-prettify": "^1.0.4",
-    "angular-sanitize": "^1.5.0",
-    "bootstrap-material-design": "^0.5.10"
+    "angular": "^1.6.2",
+    "angular-route": "^1.6.2",
+    "bootstrap": "^3.3.7",
```
Comment (jayvdb) [inline coalahtml/_coalahtml/bower.json]: this change isnt necessary, as 3.3.7 is compatible with 3.3.6

File: coalahtml/_coalahtml/bower.json [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+    "google-code-prettify": "^1.0.5",
+    "angular-sanitize": "^1.6.2",
+    "bootstrap-material-design": "^4.0.2"
```
Comment (jayvdb) [inline coalahtml/_coalahtml/bower.json]: this is the only real change in your PR.
Why do we need bootstrap-material-design ^4.0.2?

Comment (sahilarora535) [inline coalahtml/_coalahtml/bower.json]: @jayvdb I have another issue assigned to myself, which is adding pagination to the main errors page in `coala-html`. To include that, I was planning to use [`ui-bootstrap`](https://angular-ui.github.io/bootstrap/), however, the `angular version 1.4.0` is the oldest version which the tool supports, and hence decided to upgrade. For this, I used [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates). You are right, there is actually no need to upgrade bootstrap and bootstrap MD. What all should I upgrade then?

Comment (jayvdb) [inline coalahtml/_coalahtml/bower.json]: Then in your other PR, upgrade the dependencies you need upgraded, and add new ones you need.

Comment (sahilarora535) [inline coalahtml/_coalahtml/bower.json]: @jayvdb Works. Thanks! :smiley:

**Pull request closed without merging**